libs/ngxtension/navigation-end/src/navigation-end.ts

```diff
@@ -0,0 +1,22 @@
+import { Injector, inject, runInInjectionContext } from '@angular/core';
+import { Event, NavigationEnd, Router } from '@angular/router';
+import { assertInjector } from 'ngxtension/assert-injector';
+import { Observable } from 'rxjs';
+import { filter } from 'rxjs/operators';
+
+/**
+ * Creates an Observable that emits when a navigation ends.
+ * @returns An Observable of NavigationEnd events.
+ */
+export function navigationEnd(injector?: Injector): Observable<NavigationEnd> {
```

Comment:
Should we call it injectNavigationEnd ? 

Because that's what we do, we inject the router and return the navigation end. 

This way it will be understood correctly what it does. 

Just like we have injectResize. 

What do you think @nartc @va-stefanek ?

Reply:
That makes sense, let's go with It.